mengmeng/LearnCode1/Creating a New Function.swift

```diff
@@ -0,0 +1,21 @@
+func turnRight() {
+    
+    turnLeft()
+    turnLeft()
+    turnLeft()
+    
+  
+}
+
+moveForward()
+turnLeft()
```

Comment:
Lines 10-11, 12-13, 14-15, 16-17, 18-19. They’re identical. Make a func for turnRight()-moveForward() and turnLeft()-moveForward()